frontend/src/components/Modals/UploadImageModal/UploadModal.jsx

```
@@ -0,0 +1,81 @@
+                <div className={styles.uploadContainer} {...getRootProps()}>
+                    <input {...getInputProps()} />
+                    {!uploadedFile &&
+                        <>
+                            <CloudUploadOutlinedIcon style={{ border: '1px solid #ddd', padding: '0.5rem', borderRadius: '5px' }} />
+                            <p><span>Click to upload</span> or drag and drop <br></br>(maximum size: 3MB)</p>
+                        </>
+                    }
+                    {uploadedFile && <p>{uploadedFile.path}</p>}
+                </div>
+                {uploadFileErrors && <p className={styles.errorMessage}>{uploadFileErrors}</p>}
+                {fileRejections.map(({ file, errors }) => <p>{
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)